testsuite/test-depmod.c

```diff
@@ -105,8 +109,21 @@ DEFINE_TEST(depmod_another_moddir,
 	},
 	.output = {
 		.files = (const struct keyval[]) {
-			{ MODULES_ANOTHER_MODDIR_LIB_MODULES "/correct-modules.dep",
-			  MODULES_ANOTHER_MODDIR_LIB_MODULES "/modules.dep" },
+			{ MODULES_ANOTHER_MODDIR_ROOTFS "/correct-modules.dep",
+			  MODULES_ANOTHER_MODDIR_ROOTFS ANOTHER_MODDIR "/" MODULES_UNAME "/modules.dep" },
```

Comment:
See the MODULES_UNAME added here^

Reply:
Right, I missed that part due to the amount of boiler plate. Prepared some local patches to reduce that a bit.